backends/text-to-speech/windows/windows-text-to-speech.cpp

```diff
@@ -365,7 +365,7 @@ void WindowsTextToSpeechManager::createVoice(void *cpVoiceToken) {
 	}
 
 	if (desc == "Sample TTS Voice") {
-		// This is really bad voice, it is basicaly unusable
+		// This is really a bad voice, it is basically unusable
```

Comment:
As we are already making petty changes here, I'd suggest rewording this to "a really bad voice". That sounds more natural to me.

Reply:
Everything to please the crowd